src/website/layouts/partials/sidenav.pug

```diff
@@ -11,6 +11,34 @@ ul.chi-tabs.-vertical.-lg.-animated
                 if section.disabled
                   //- li
                   //-   a(href=`${rootPath}${section.path}`, disabled="disabled") #{section.title}
+                else if section.accessibilityAccordion
+                  li
+                    .chi-accordion.accessibility-accordion
+                      .chi-accordion__item.-expanded
+                        button.chi-accordion__trigger
+                          .chi-accordion__title #{section.title}
+                          i.chi-icon.icon-chevron-down
+                        .chi-accordion__content.-p--0
+                          ul.chi-tabs__subtabs
+                            li(class=`${'foundations/overview' === path ? "-active" : ""}`)
```

Comment:
Let's change the paths for all accessibility pages to include accessibility. e.g. `${rootPath}foundations/accessibility/overview`
`${rootPath}foundations/accessibility/using-keyboard`
`${rootPath}foundations/accessibility/page-structure`